bindings/resolve.go

```diff
@@ -30,25 +30,32 @@ type Predicate func(bind libcnb.Binding) bool
 // case-insensitive.
 func OfType(t string) Predicate {
 	return func(bind libcnb.Binding) bool {
-		return strings.ToLower(bind.Type) == strings.ToLower(t)
+		return strings.EqualFold(bind.Type, t)
```

Comment:
I'm curious about this change. Are you seeing specific issues with `ToLower`? The behavior isn't the same, so wanting to understand more.

Reply:
It was just some issue raised by a linter in the `IDE`.

> strings.EqualFold() Function in Golang reports whether s and t, interpreted as UTF-8 strings, are equal under Unicode case-folding, which is a more general form of case-insensitivity. 

For non UTF-8, it should be the same.